Source/Core/DolphinWX/MainMenuBar.cpp

```diff
@@ -423,6 +423,8 @@ wxMenu* MainMenuBar::CreateSymbolsMenu() const
       IDM_SCAN_SIGNATURES, _("&Signature Database"),
       _("Recognise standard functions from Sys/totaldb.dsy, and use generic zz_ "
         "names for other functions."));
+  generate_symbols_menu->Append(IDM_SCAN_RSO, _("&RSO Modules"),
```

Comment:
Noticed this just now, but note that `&Rename Symbols from File...` is already using `&R` as its accelerator and is within the parent menu to the one `generate_symbols_menu` is within.